api/Services.Sql/Reports/BriefResponseService.cs

```diff
@@ -0,0 +1,33 @@
+using Microsoft.EntityFrameworkCore;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using System.Text.Json;
+using Dapper;
+using Dta.Marketplace.Api.Services.Reports;
+
+namespace Dta.Marketplace.Api.Services.Sql.Reports {
+    public class BriefResponseService : IBriefResponseService {
+        private readonly DigitalMarketplaceContext _context;
+
+        public BriefResponseService(DigitalMarketplaceContext context) {
+            _context = context;
+        }
+        public async Task<IEnumerable<dynamic>> GetSubmittedBriefResponsesAsync() {
+            var connection = _context.Database.GetDbConnection();
+            return await connection.QueryAsync<dynamic>(
+                sql: @"
+                    SELECT
+                        br.brief_id,
+                        br.supplier_code,
+                        br.created_at,
+                        br.data ->> 'dayRate' AS dayRate,
```

Comment:
i hate keeping the old format
```suggestion
                        br.data ->> 'dayRate' AS day_rate,
```